empty-plugin/src/main/resources/archetype-resources/.gitignore

```diff
@@ -0,0 +1,2 @@
+target
+work
```

Comment:
How about excluding project files from IDEs too? Like https://github.com/jenkinsci/buildtriggerbadge-plugin/blob/master/.gitignore#L4-L13 (missing netbeans for now, not sure how/if/where it stores it)

Reply:
NetBeans does not have any mandatory IDE-specific files for Maven projects—`pom.xml` suffices for most purposes.

Anyway, yes it is a good idea to add Eclipse- and IDEA-specific ignores. I do not use either of those so I will just trust that they are accurate.

Also I will add some entries from https://github.com/github/gitignore/blob/master/Maven.gitignore that are potentially applicable to Jenkins plugins, such as `pom.xml.releaseBackup`.